orc8r/protos/mconfig/mconfigs.proto

```diff
@@ -18,6 +18,38 @@ import "orc8r/protos/common.proto";
 package magma.mconfig;
 option go_package = "magma/orc8r/lib/go/protos/mconfig";
 
+
+//===========================================================================
+// SharedMConfig is a wrapper around configs that are shared amongst services
+//===========================================================================
+// --------------------------------------------------------------------------
+// SentryConfig stores the network-wide Sentry.io configuration
+// TODO(hcgatewood): rename, duplicated in lte/.../mconfigs
+// --------------------------------------------------------------------------
+message SharedSentryConfig {
+  // url_python initializes the Sentry Python SDK and sets the remote URL.
+  // If set to empty string, Sentry Python SDK will not be initialized.
+  string url_python = 1;
+  // url_native initializes the Sentry Native SDK for C/C++ and sets the
+  // remote URL. If set to empty string, Sentry Native SDK will not be
+  // initialized.
+  string url_native = 2;
+  // upload_mme_log decides whether MME service log file (/var/log/mme.log)
+  // is uploaded along with MME crashreports
+  bool upload_mme_log = 3;
+  // sample_rate sets the rate at which Python error events are sampled.
+  // sample_rate should be a number between 0 (0% of errors sent) and 1 (100%
+  // of errors sent)
+  float sample_rate = 4;
+}
+
+// --------------------------------------------------------------------------
+// SharedMConfig is a wrapper around configs that are shared amongst services
+// --------------------------------------------------------------------------
+message SharedMConfig {
```

Comment:
Nit: structure files top-down -- i.e. as you jump-to-definition, you should be moving progressively down the file (increasing line number)

So that to say, place sharedmconfig above sharedsentryconfig